package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,22 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = [line[6:15], line[15:24],
+                                  line[24:33], line[33:40],
+                                  line[40:47], line[47:54]]
+                    if (np.array(cell_dims,dtype=float) == 
+                        np.array([1,1,1,90,90,90],dtype=float)).all():
+                        warnings.warn("1 A^3 CRYST1 record," 
+                                      "possibly an EM structure file, setting box-size to 0",stacklevel=2)
+                        self.ts._unitcell=np.array([0,0,0,90,90,90],dtype=float)    
                 except ValueError:
                     warnings.warn("Failed to read CRYST1 record, "
                                   "possibly invalid PDB file, got:\n{}"
                                   "".format(line))
-
+                else:
+                    if not (self.ts._unitcell==np.array([0,0,0,90,90,90])).all():
```

Comment:
See above, I would be expecting most of what is currently in `try` (except assigning `cell_dims`) to be here.

Reply:
Do let me know if it has been done the right way in my latest commit.